### PR TITLE
Full-audit-ID-691

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-6e14dad9b9/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f2b073efa8/how-to/)

File: content/general/update-an-existing-ssl-certificate.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 last_modified_by: Cat Lookabaugh
 product: undefined
 product_url: undefined
 ---
 
-Updating your existing SSL Certificate is a quick and simple process.
-Whether your SSL Certificate is expiring or you are simply needing to
-change to a new SSL Certificate, you can follow these instructions to
+Updating your existing SSL certificate is a quick process.
+Whether your SSL certificate is expiring or you are simply needing to
```
Comment (stephamon) [inline content/general/update-an-existing-ssl-certificate.md]: ...or you simply need to change...

File: content/general/update-an-existing-ssl-certificate.md [hunk 2]
```diff
@@ -22,16 +22,15 @@ article posted in the Rackspace Support forum.
 
 ### Getting Started
 
-In order to update or renew your SSL Certificate, you will need your
+In order to update or renew your SSL certificate, you need your
 CSR. If you don't have your CSR saved, please contact support to have a
```
Comment (stephamon) [inline content/general/update-an-existing-ssl-certificate.md]: ...your CSR saved, contact support to have...

File: content/general/update-an-existing-ssl-certificate.md [hunk 3]
```diff
@@ -22,16 +22,15 @@ article posted in the Rackspace Support forum.
...
-ticket, live chat or via phone and we can assist you with obtaining this
-information.
+new one generated. You can contact our **Fanatical Support** through a
+ticket, live chat, or a phone call, and we can help you.
 
 ### Renewing Your Certificate
 
-Next, you will need to update/renew your SSL Certificate. SSL
-Certificates are available from a number of third party sources. Some
+Next, you need to update/renew your SSL certificate. SSL
```
Comment (stephamon) [inline content/general/update-an-existing-ssl-certificate.md]: update or renew

File: content/general/update-an-existing-ssl-certificate.md [hunk 4]
```diff
@@ -51,25 +50,25 @@ Once you have completed your certificate renewal, you're ready to install the ne
...
+Once you have this information, you can install your new
+certificate by clicking on the **Security** tab of your site, then clicking on
 the **Edit Certificate link**.
 
-On the next screen, you will see three fields for Certificate, Private
-Key, and Intermediate Certificate; simply replace the current
-information with your updated info and hit save.
+On the next screen, there are three fields for **Certificate**, **Private
+Key**, and **Intermediate Certificate**. Replace the current
+information with your updated info, and hit save.
```
Comment (stephamon) [inline content/general/update-an-existing-ssl-certificate.md]: ...and then click **Save**.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-01c9933714/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**